### PR TITLE
Fix Map Landing Lights

### PR DESCRIPTION
**Pull request by Steelpoint**
# About the pull request
This PR fixes issues with landing lights on the following maps.
- LV-624
- Solaris Ridge
- Fiorina Science Annex
- Trijent Dam
- Sorokyne Strata
- Kutjevo Refinery

This PR also adds landing lights for LZ2 on Shivas Snowball. LZ1 does not gain any due to its 'non standard landing' nature. For Lv-624, the lower two floodlights have been pushed one tile away from the landing zone for consistency.

Fixes Issue #3427 

# Explain why it's good for the game

Fixes glaring issues with landing lights on many maps being out of order, not working or flashing in a odd way. Addition of landing lights to LZ2 of Shivas Snowball brings it up to standard with every other LZ. 

# Testing Photographs and Procedure
New LZ 1
New LZ1 appearance. 

# Changelog
:cl:
maptweak: Fixed landing strip lights on most maps
/:cl:

### CODE REVIEW

**Comment by Steelpoint** [comment]
First PR in 5 years so be gentle.

**Review by Drulikar — CHANGES_REQUESTED** [review]
Much better, though many of the landing delays sporadically repeat (as in two delayones next to eachother: e.g. center left for LZ1 soro, and top right for LZ2 soro). It's easiest to identify this problem by using the type filters in strongdmm to only highlight each type individually.
image
image
image
image
etc.

**Comment by Steelpoint** [comment]
I think some of those are due to the landing zones not being big enough for the light pattern to be completed (goes ds1 > 3 > 2 > 1 > repeat), I'll double check the maps to ensure they are as good as I can make.

**Comment by Steelpoint** [comment]
I've done a second pass, I believe I've fixed the remaining outstanding issues.

I did effect a small change to the LZ's on Desert Dam. Decreasing the LZ size by 1 tile in length at the top.

Screenshot 2023-06-06 12 20 30
LZ1

Screenshot 2023-06-06 12 20 25
LZ2

File: maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm
Comment (Drulikar) [inline maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm:1]: LZ1 is fine.
LZ2 has two errors: 
Fiorina LZ2

**Review by Drulikar — CHANGES_REQUESTED** [review]
LV624, Kutjevo, and Desert Dam should be good now.

**Comment by Steelpoint** [comment]
Fixed the outstanding issues.

**Review by Drulikar — APPROVED** [review]

**Pull request merged**